### DISCUSSION TITLE
Здраствуйте господа, обнаружил некоторую проблему работ приложения с Zapret

### DISCUSSION THREAD

**Discussion by Lonrion** [Flowseal/zapret-discord-youtube#833] [Q&A]
Если быть конкретнее. Клиент Riot(League of legends) очень отвратно робит вместе с включенным zapret. В Hunt showdown меня кикает из лобби, что менее критично, но в условную лигу  я просто не могу сыграть ибо клиент просто не позволяет принять матч

**Comment by cinema4d** [comment]
Такая же штука происходит при заходе в игру, она закрывается думая что это чит
